src/static/current-state/components/QueueItem.tsx

```diff
@@ -259,11 +258,22 @@ export class QueueItem extends React.Component<QueueItemProps, QueueItemState> {
             </StatusItem>
           ))}
         </div>
-        {status.reason ? (
+        {status.reason && status.state !== 'success' ? (
```

Comment:
What was the reason behind this change?

Reply:
My reasoning behind this was because the "Landed by:" field I added has the name of the triggerer so it wouldn't make sense to display the reason too in this case since the reason would contain the name too if the state is `success`.